scalableminds/webknossos
# In Transfer Tasks Modal, show User Name

### URL of deployed dev instance (used for testing):
- https://transfertasksusername.webknossos.xyz

### Steps to test:
- create some tasks, request, leave open
- in admin project list view, click on “show active users”
- modal should list names next to email adresses

![image](https://user-images.githubusercontent.com/1390035/53016632-55051280-344e-11e9-8831-43cc3114a810.png)


### Issues:
- fixes #3793

------
